### DISCUSSION TITLE
SnapClientinternalDAC

### DISCUSSION THREAD

**Discussion by parabyte** [pschatzmann/arduino-snapclient#31] [Q&A] [answered]
im using your example sketch with these changes

WAVDecoder pcm;
AnalogAudioStream out;
WiFiClient wifi;
SnapTimeSyncDynamic synch(172, 10); // optional configuratioin
SnapClient client(wifi, out, pcm);

I am getting no audio output, if i use opus and other codec's i do indeed have audio,

Your library does sample rate conversion?

**Comment by pschatzmann** [comment, marked as answer]
Just follow the [documentation](https://github.com/pschatzmann/arduino-snapclient/wiki/Selecting-a-Decoder)

**Comment by parabyte** [comment]
i think there is a typo in the documentation, in the pcm section, SnapClient client(wifi, out, opus);

**Reply by pschatzmann** [reply]
Confirmed: it has been corrected...
